src/senselab/text/tasks/emotional_analysis/api.py

```diff
@@ -0,0 +1,33 @@
+"""API module for emtoional analysis."""
+
+from typing import Dict, List, Optional, Union
+
+from senselab.text.tasks.emotional_analysis.emotional_analysis import EmotionalAnalysis
+from senselab.utils.data_structures.device import DeviceType
+from senselab.utils.data_structures.model import HFModel
+
+
+def analyze_emotion(
+    pieces_of_text: List[str],
+    device: Optional[DeviceType] = None,
+    model: Optional[HFModel] = None,
```

Comment:
I think model should be of type `SenselabModel`. this way, we can accept models that are not necessarily HF models